cli/tools/lint.rs

```diff
@@ -144,13 +203,24 @@ pub fn create_linter(syntax: Syntax, rules: Vec<Box<dyn LintRule>>) -> Linter {
 
 fn lint_file(
   file_path: PathBuf,
+  maybe_lint_config: Option<&LintConfig>,
+  rules_tags: Vec<String>,
+  rules_include: Vec<String>,
+  rules_exclude: Vec<String>,
 ) -> Result<(Vec<LintDiagnostic>, String), AnyError> {
   let file_name = file_path.to_string_lossy().to_string();
   let source_code = fs::read_to_string(&file_path)?;
   let media_type = MediaType::from(&file_path);
   let syntax = ast::get_syntax(&media_type);
 
-  let lint_rules = rules::get_recommended_rules();
+  // Obtaining rules from config is infallible at this point.
+  let lint_rules = get_configured_rules(
+    maybe_lint_config,
+    rules_tags,
+    rules_include,
+    rules_exclude,
+  )
+  .unwrap();
```

Comment:
Is this comment true? If the code path gets into

```rust
  if configured_rules.is_empty() {
    anyhow!("No rules have been configured");
  }
```

in `get_configured_rules` function because the user has specified all rules as "exclude", then returned value will be `Err`, no?

Reply:
explanation: https://github.com/denoland/deno/pull/11776#discussion_r700217088